src/components/app/__snapshots__/App.test.tsx.snap

```diff
@@ -0,0 +1,474 @@
+              height="48"
+              viewBox="0 0 48 48"
+              width="48"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <g
+                id="Direction=Down"
+              >
+                <path
+                  clip-rule="evenodd"
+                  d="M8.44365 14.2011L24 29.7574L39.5563 14.2011L41.6738 16.3272L24 34.0001L6.32366 16.3237L8.44365 14.2011Z"
+                  fill="#fff"
+                  fill-rule="evenodd"
+                  id="Union"
+                />
+              </g>
+            </svg>
+          </li>
+        </ul>
+      </div>
+    </header>
+    <main
+      class="App"
+    >
+      <div
+        class="_hero_8a1d90"
+      >
+        <section
+          class="_infoSection_8a1d90"
+        >
+          <h1
+            class="_title_8a1d90"
+          >
+            Welcome back to Motorway
+          </h1>
+          <h2
+            class="_subtitle_8a1d90"
+          >
+            The worlds best online car image library taken by professional photographers
+          </h2>
+        </section>
+        <section
+          class="_imageSection_8a1d90"
+        >
+          <img
+            alt=""
+            src="images/hero.png"
+          />
+        </section>
+      </div>
+      <div
+        class="Task2"
+      />
+    </main>
+  </div>,
+  "debug": [Function],
+  "findAllByAltText": [Function],
+  "findAllByDisplayValue": [Function],
+  "findAllByLabelText": [Function],
+  "findAllByPlaceholderText": [Function],
+  "findAllByRole": [Function],
+  "findAllByTestId": [Function],
+  "findAllByText": [Function],
+  "findAllByTitle": [Function],
+  "findByAltText": [Function],
+  "findByDisplayValue": [Function],
+  "findByLabelText": [Function],
+  "findByPlaceholderText": [Function],
+  "findByRole": [Function],
+  "findByTestId": [Function],
+  "findByText": [Function],
+  "findByTitle": [Function],
+  "getAllByAltText": [Function],
+  "getAllByDisplayValue": [Function],
+  "getAllByLabelText": [Function],
+  "getAllByPlaceholderText": [Function],
+  "getAllByRole": [Function],
+  "getAllByTestId": [Function],
+  "getAllByText": [Function],
+  "getAllByTitle": [Function],
+  "getByAltText": [Function],
+  "getByDisplayValue": [Function],
+  "getByLabelText": [Function],
+  "getByPlaceholderText": [Function],
+  "getByRole": [Function],
+  "getByTestId": [Function],
+  "getByText": [Function],
+  "getByTitle": [Function],
+  "queryAllByAltText": [Function],
+  "queryAllByDisplayValue": [Function],
+  "queryAllByLabelText": [Function],
+  "queryAllByPlaceholderText": [Function],
+  "queryAllByRole": [Function],
+  "queryAllByTestId": [Function],
+  "queryAllByText": [Function],
+  "queryAllByTitle": [Function],
+  "queryByAltText": [Function],
+  "queryByDisplayValue": [Function],
+  "queryByLabelText": [Function],
+  "queryByPlaceholderText": [Function],
+  "queryByRole": [Function],
+  "queryByTestId": [Function],
+  "queryByText": [Function],
```

Comment:
hey @kml1990, we are still capturing all those mock fns which arent usually what you want